src/wizard/workflow.scss

```diff
@@ -2,49 +2,37 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-wizards-workflow {
-  @include iui-wizards;
-
-  > .iui-wizards-wrapper {
+@mixin iui-wizard-workflow {
+  ol {
     align-items: center;
+  }
 
-    > .iui-wizards-step {
-      width: auto;
-
-      > .iui-wizards-step-circle {
-        padding-left: $iui-m;
-        padding-right: $iui-m;
-        border-radius: $iui-sm;
-        width: auto;
-        height: auto;
-        min-height: $iui-l;
-        @include themed {
-          border-color: t(iui-color-foreground-positive);
-          color: t(iui-color-foreground-positive);
-        }
-      }
-
-      &.iui-wizards-step-current > .iui-wizards-step-circle {
-        border-radius: 0;
-        font-weight: $iui-font-weight-normal;
-        @include themed {
-          color: t(iui-color-foreground-accessory);
-          border-color: t(iui-color-background-positive);
-        }
-      }
+  .iui-wizard-circle {
+    padding-left: $iui-m;
+    padding-right: $iui-m;
+    border-radius: $iui-sm;
+    word-break: break-word;
```

Comment:
Is this needed? I see we already have `overflow-wrap: break-word` in the base wizard circle

Reply:
I don't believe this is needed, removed in next push.